spec/police_spec.rb

```diff
@@ -93,11 +93,12 @@
   context 'when police is called' do
     it 'travels to active neighborhood' do
       police.travel_to(neighborhood)
-      expect(police.active_neighborhoods.last).to be(neighborhood)
+      expect(police.active_neighborhoods.last).to be_instance_of(Neighborhood)
```

Comment:
Dabar šis testas tikrina kitą dalyką.

**Anksčiau:** 
Tikrino ar policija nuvyko į numatytąjį rajoną.

**Dabar:** 
Tikrina ar tiesiog kažkur nuvyko.

Toks ir buvo tavo tikslas?